src/encoded/types/biosample.py

```diff
@@ -513,6 +513,24 @@ def summary(self, request,
         return construct_biosample_summary([biosample_dictionary],
                                            sentence_parts)
 
+    @calculated_property(schema={
+        "title": "Perturbed",
+        "description": "A flag to indicate whether the biosample has been perturbed with a treatment or genetic modification.",
+        "type": "boolean",
+        "notSubmittable": True,
+    })
+    def perturbed(
+        self,
+        request,
+        applied_modifications,
+        treatments=None,
+    ):
+        return bool(treatments) or any(
+            (
+                request.embed(m, '@@object').get('perturbation', False) for m in applied_modifications
```

Comment:
Minor formatting nit. Usually worth splitting up comprehensions into multiple lines, where thing getting returned is first line, iterator is second line, and (when applicable) if statement is third line:

```
[
    d.get('x')
    for d in data
    if some_condition
]
```
Probably not worth changing here though.

Reply:
I can fix it, it should be a quick change.